entrypoint/certrotate/certrotate_suite_test.go

```diff
@@ -41,14 +45,30 @@ func TestMain(m *testing.M) {
 }
 
 func TestCertRotateSuite(t *testing.T) {
-	RegisterFailHandler(Fail)
+	RegisterFailHandler(FailWithReport)
 	RunSpecs(t, "Certificate Rotate Test Suite")
 }
 
+var _ = ReportAfterSuite("Certificate Rotate Test Suite", func(report Report) {
+	// Add Qase reporting capabilities.
+	if qaseReport == "true" {
```

Comment:
`strings.ToLower` here and all other places

Reply:
tks